geomesa-accumulo/geomesa-accumulo-tools/conf-filtered/accumulo-env.sh

```diff
@@ -54,3 +54,5 @@ function get_accumulo_classpath() {
     echo "$accumulo_cp" | sed 's/[^:]*geomesa-accumulo-distributed-runtime[^:]*jar//'
   fi
 }
+
+get_accumulo_classpath
```

Comment:
What does adding this do?

Reply:
this makes it so that when you source the script it will print out the classpath (vs having to call the function after sourcing the script). this was done so that we can just source all the *-env.sh files in the conf folder in order to build up the classpath without having to specify them per back-end, which means we can use the same geomesa-* script for all of them.